sample/firehose-LocationHours-3-records-encoded.json

```diff
@@ -1,45 +1,44 @@
 {
-    "invocationId": "invocationIdExample",
-    "deliveryStreamArn": "arn:aws:kinesis:deliveryStream/LocationHours-qa",
-    "sourceKinesisStreamArn": "arn:aws:kinesis:us-east-1:946183545209:stream/LocationHours-qa",
-    "region": "us-east-1",
-    "records": [
-      {
-        "recordId": "49546986683135544286507457936321625675700192471156785154",
-        "approximateArrivalTimestamp": 1495072949453,
-        "kinesisRecordMetadata": {
-          "sequenceNumber": "49545115243490985018280067714973144582180062593244200961",
-          "subsequenceNumber": "123456",
-          "partitionKey": "partitionKey-03",
-          "shardId": "shardId-000000000000",
-          "approximateArrivalTimestamp": 1495072949453
-        },
-        "data": "AARhYQISTGlicmFyeSBBAAZNb24CCjA5OjAwAgoxNzowMAIUMjAyMy0wMS0wMQ=="
+  "invocationId": "invocationIdExample",
+  "deliveryStreamArn": "arn:aws:kinesis:deliveryStream/LocationHours-qa",
+  "sourceKinesisStreamArn": "arn:aws:kinesis:us-east-1:946183545209:stream/LocationHours-qa",
```

Comment:
Sorry I didn't catch this before but we shouldn't be using our real AWS account id here and in the other encoded records. Replace them all with mock ids.

Reply:
Do you mean the ID in the `sourceKinesisStreamArn`? I removed that ID but lmk if there's anything else

Reply:
Yep that's what I meant -- thanks!